### ISSUE TITLE
XNAT mail is broken because of a spam filter on smtp.camh.ca

### ISSUE THREAD

**Issue by josephdviviano** [TIGRLab/admin#1326]
`smtp.camh.ca` can send email to internal addresses:

```
telnet smtp.camh.net 25
```

**Comment by josephdviviano** [comment]
Just tested, `info2.camh.net` will send to external addresses.

**Comment by josephdviviano** [comment]
However, xnat still does not send emails.

**Comment by josephdviviano** [comment]
I think someone really did a number on the XNAT configuration -- I've found logs in at least 4 different locations.

I'm almost inclined to ditch this all together and just upgrade to XNAT 1.7.

**Comment by DESm1th** [comment]
Yeah I noticed that too. I think part of it may have been Tom to help with diagnosing that serious issue we had with the XNAT server grinding to a halt ages ago and then it just never got restored to normal. Although if I recall I think even before that XNAT had logs in weird places...?
